packages/client-core/src/user/VideoWindows/index.tsx

```diff
@@ -137,7 +137,7 @@ export const useMediaWindows = () => {
     mediaNetwork.users?.[selfUserID] &&
     !windows.find(({ peerID }) => mediaNetwork.users[selfUserID]?.includes(peerID))
   ) {
-    windows.unshift({ peerID: selfPeerID, type: 'cam' })
+    windows.unshift({ peerID: selfPeerID, userID: selfUser.id.value, type: 'cam' })
```

Comment:
It looks like this gets the userId for `selfUser` but it isnt getting the user ids for the other users.

I think you'll need to get the all the userIDs by peerID. I found this example code that could work to get them.

```
mediaNetwork?.peers?.[peerID]?.userId
```

https://github.com/ir-engine/ir-engine/blob/dev/packages/client-core/src/user/VideoWindows/hook.tsx#L429